Docs/Configuration.tex

```diff
@@ -6828,6 +6828,51 @@ \subsection{AppleInput Properties}\label{uefiappleinputprops}
   \item \texttt{false} --- Prevent key input mirroring to non-Apple protocols when in graphics mode.
   \end{itemize}
 
+  \item
+  \texttt{PointerPollMin}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{0}\\
+  \textbf{Description}: Configure minimal pointer polling period in ms.
```

Comment:
Do we have any convention on "minimal" vs "minimum" (see "maximum" below)?

Reply:
I believe not. We use Minimum in the options, but otherwise choose whatever fits best in the context. I think I will leave it unchanged here.